### DISCUSSION TITLE
Running locally on windows and setting up firebase

### DISCUSSION THREAD

**Discussion by muhidabid** [ruxailab/RUXAILAB#356] [Q&A] [answered]
Hi, I am trying to run this project on windows. I am having an issue setting up firebase. I created a firebase project and followed instructions in [this discussion](https://github.com/orgs/ruxailab/discussions/331) and my project runs fine upon `npm serve`. However, when it comes to signing in after signing up, it says incorrect username or password. 

![image](https://github.com/ruxailab/RUXAILAB/assets/82958857/37f41f02-e913-4f46-b353-6c2f66e310ec)

I understand I have to set up the firebase emulator but after upon writing the command `firebase emulators:start` I am faced with this error:

![image](https://github.com/ruxailab/RUXAILAB/assets/82958857/e92c36b6-293b-4996-b6d9-4dc665a8ee1b)

**Comment by jvJUCA** [comment, marked as answer]
Dear @muhidabid,

To ensure you have all the necessary instructions to address this issue, I would recommend you to review this discussion: [discussion of this run issues](https://github.com/ruxailab/RUXAILAB/discussions/309), and also watch our tutorial video on how to run the project.

[Link to the tutorial video - "Tutorial - Running RUXAILAB with Firebase Emulators 🖥️"](https://youtu.be/dAf4LRxITCc)

Best regards,

@jvJUCA

**Comment by KarinePistili** [comment]
I'll be closing this discussion as conversations are being done in discussion #309
